auth/README.md

```diff
@@ -319,6 +320,45 @@ IdpConfig phoneConfigWithDefaultNumber = new IdpConfig.PhoneBuilder()
         .build();
 ```
 
+###### Limiting the list of available countries in the country selector
+
+You can limit the countries shown in the country selector list. By default, all countries are shown. 
+
+You can provide a list of countries to whitelist or blacklist. You can populate these lists with 
+ISO (alpha-2) and E164 formatted country codes. 
+
+```java
+List<String> whitelistedCountries = new ArrayList<String>();
+whitelistedCountries.add("+1");
+whitelistedCountries.add("+gr");
```

Comment:
Is `+gr` correct or a typo?  I thought `+` goes before the numeric version only?

Reply:
Good catch. It's a typo, removed it.